apps/address-service/domains/common/utils/services/search/plugins/SearchByInjectionId.js

```diff
@@ -0,0 +1,76 @@
+const { AbstractSearchPlugin } = require('@address-service/domains/common/utils/services/search/AbstractSearchPlugin')
+const { Address } = require('@address-service/domains/address/utils/serverSchema')
+const { InjectionsSeeker } = require('@address-service/domains/common/utils/services/InjectionsSeeker')
+const { AddressInjection } = require('@address-service/domains/address/utils/serverSchema')
+const get = require('lodash/get')
+const { generateAddressKey } = require('@address-service/domains/common/utils/addressKeyUtils')
+const { INJECTIONS_PROVIDER } = require('@address-service/domains/common/constants/providers')
+
+const SEPARATOR = ':'
+const UUID_REGEX = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/
```

Comment:
I suppose you can use `import { validate } from 'uuid'` for that

Reply:
Refactored. Thanks for the suggestion :)